src/common/src/config.rs

```diff
@@ -147,6 +144,9 @@ pub struct StorageConfig {
     #[serde(default = "default::data_directory")]
     pub data_directory: String,
 
+    #[serde(default = "default::s3_prefix_bytes_len")]
+    pub s3_prefix_bytes_len: u32,
```

Comment:
Can you add more documentations to describe what this config means and how it relates to the number of S3 prefixes used in this cluster?